src/Lumi/Components2/Banner.purs

```diff
@@ -0,0 +1,231 @@
+  , content :: Array JSX
+  )
+
+banner :: LumiComponent BannerProps
+banner =
+  unsafePerformEffect do
+    lumiComponent "Banner" defaults \props -> React.do
+      theme <- useTheme
+      visible /\ setVisible <- useState true
+      pure
+        $ Monoid.guard visible
+        $ box
+        $ style props.css
+        $ S.banner
+        $ _ { className = props.className
+            , content =
+                [ box
+                  $ S._column
+                  $ S._alignSelf S.Center
+                  $ style_
+                      ( css
+                        { flex: str "1", width: str "100%" }
+                      )
+                  $ _ { content =
+                          [ case props.title of
+                              Just title ->
+                                box
+                                $ S._alignSelf S.Start
+                                $ style_
+                                    ( css
+                                      { marginBottom: int 8
+                                      , maxWidth: str "calc(100% - 24px)"
+                                      }
+                                    )
+                                $ _ { content = [ title ]
+                                    }
+                              _ -> mempty
+                          , box
+                            $ S._align S.Center
+                            $ S._row
+                            $ style_
+                                ( css
+                                  { flex: str "1" }
+                                )
+                            $ _ { content =
+                                    [ case props.icon of
+                                        Just icon ->
+                                          box
+                                          $ S._alignSelf S.Start
+                                          $ S.onDesktop (S._alignSelf S.Center)
+                                          $ style_
+                                              ( fold
+                                                  [ css
+                                                      { marginRight: int 16
+                                                      }
+                                                  , desktopQuery $ css
+                                                      { alignSelf: str "center"
+                                                      }
+                                                  ]
+                                              )
+                                          $ _ { content = [ icon ]
+                                              }
+                                        _ -> mempty
+                                    ]
+                                    <> props.content
+                                }
+                          ]
+                      }
+                ,  Monoid.guard props.dismissable
+                    $ E.element R.div'
```

Comment:
I think we should use a `button` element here for proper accessibility, maybe with `aria-label: "dismiss"` too? Note that this implies in adding `outline: none; border: none; background: none` to `dismissButtonStyle`.